CRM/Upgrade/Incremental/php/FiveFortyOne.php

```diff
@@ -83,6 +83,33 @@ public function upgrade_5_41_alpha1($rev) {
     $this->addTask('Replace contribution page id token in action schedule',
       'updateActionScheduleToken', 'contribution.contribution_page_id', 'contribution.contribution_page_id:label', $rev
     );
+    $this->addTask('Ensure multilingual installs have a default financial type for the membership price set.',
+      'ensureDefaultMembershipFinancialType');
+  }
+
+  public function ensureDefaultMembershipFinancialType(): bool {
+    $membershipPriceSet = CRM_Core_DAO::executeQuery(
+      'SELECT id, financial_type_id FROM civicrm_price_set WHERE name = "default_membership_type_amount"'
+    );
+    $membershipPriceSet->fetch();
+    if (!is_numeric($membershipPriceSet->financial_type_id)) {
+      $membershipFinancialTypeID = CRM_Core_DAO::singleValueQuery('
+        SELECT id FROM civicrm_membership WHERE name = "Member Dues"
+        OR name = " ' . ts('Member Dues') . '"
+      ');
+      if (!$membershipFinancialTypeID) {
+        // This should be unreachable - but something is better than nothing
+        // if we get to this point & 2 will be correct on 99.9% of installs.
+        $membershipFinancialTypeID = 2;
+      }
+      CRM_Core_DAO::executeQuery("
+        UPDATE civicrm_price_set
+        SET financial_type_id = $membershipFinancialTypeID
+        WHERE name = default_membership_type_amount"
```

Comment:
quotes missing around `default_membership_type_amount`

Reply:
yeah - you can tell I' hadn't actually r-run it...